### PR TITLE
Master f2fs fix

### PR DESCRIPTION
**Pull request by leon-anavi**
# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [X] Make sure that all commits have a [`Changelog`](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md#changelog-tags) tag. If nothing should be added to the Changelog, add a `Changelog: None` tag. If there is a change, add `Changelog: Commit`, or `Changelog: Title`, depending on what should be included in the changelog.

- [X] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

This pull request brings a couple of improvements and bug fixes related to F2FS:

* Add a dependency to f2fs-tools-native in `mender-part-images` if using F2FS for the data partition.
* Load Mender configuration files to the data partition when using F2FS through `sload.f2fs` in class `dataimg`.

### CODE REVIEW

File: meta-mender-core/classes/mender-dataimg.bbclass [hunk 1]
```diff
@@ -4,25 +4,30 @@
...
     fi
 
     dd if=/dev/zero of="${WORKDIR}/data.${MENDER_DATA_PART_FSTYPE_TO_GEN}" count=0 bs=1M seek=${MENDER_DATA_PART_SIZE_MB}
     mkfs.${MENDER_DATA_PART_FSTYPE_TO_GEN} \
         $force_flag \
         "${WORKDIR}/data.${MENDER_DATA_PART_FSTYPE_TO_GEN}" \
-        $root_dir_flag "${IMAGE_ROOTFS}/data" \
-        $volume_label_flag "${MENDER_DATA_PART_LABEL}" \
+        $root_dir_flag \
+        $volume_label_flag data \
```
Comment (kacf) [inline meta-mender-core/classes/mender-dataimg.bbclass:24]: Why remove `${MENDER_DATA_PART_LABEL}`?

**Review by kacf — CHANGES_REQUESTED** [review]
I'd change to `Changelog: Commit` on both of these commits, since they are bug fixes.

**Comment by leon-anavi** [comment]
Hi @kacf,

> I'd change to `Changelog: Commit` on both of these commits, since they are bug fixes.

Following your feedback I have made the required modifications in the code and the git commit message. Thank you for the code review. Is the new version of the pull request OK?

Thanks,
Leon

**Review by kacf — APPROVED** [review]
Yes, thank you! I'll run this through our full test suite now.

**Comment by kacf** [comment]
Just spurious failures.

**Pull request merged**